src/styles/box.scss

```diff
@@ -0,0 +1,31 @@
+.box {
+  display: grid;
+  grid-template-columns: repeat(3, minmax($size, 1fr));
+  height: 100vh;
+
+  @media (max-width: 899px) {
+    grid-template-columns: repeat(2, 1fr);
+    grid-template-rows: repeat(2, 1fr);
+  }
+
+  @media (max-width: 600px) {
```

Comment:
that's what I see on 600px breakpoint, it should be 2 blocks, hint: you should lower max-width value a bit
<img width="606" alt="image" src="https://user-images.githubusercontent.com/37161315/157496525-3d9f71ae-5b68-4927-bb8a-bc2cf54e1742.png">